### PR TITLE
LB-123: Setup Docker for MessyBrainz

### PR DESCRIPTION
**Pull request by paramsingh**
Almost all of the stuff taken from LB. Had to fix some tests as well.

### CODE REVIEW

**Review by mayhem — APPROVED** [review]
Looks good to me. Unless Alastair has comments, lets merge this and then do LB-160.

**Comment by paramsingh** [comment]
I have the LB-160 patch ready to send as soon as this gets merged.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM python:2.7.12
```
Comment (alastair) [inline Dockerfile:1]: I guess the idea is that we switch this to python3 once this PR is merged?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,26 @@ commands:
...
+
+    $ docker-compose -f docker/docker-compose.yml -p messybrainz run --rm web bash -c "python manage.py init_db"
+
+Everything should be good to go now. You should be able to access the webserver at `http://localhost:8080`.
+
+Also, in order to run the tests, just use the command: `./test.sh`.
+
 ### The Usual Way
 
 Full installation instructions are available in [INSTALL.md](https://github.com/metabrainz/messybrainz-server/blob/master/INSTALL.md) file.
```
Comment (alastair) [inline README.md]: Do we need to have normal installation instructions? I'm happy to force everyone to use Docker

File: test.sh [hunk 3]
```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```
Comment (alastair) [inline test.sh:1]: We're now sharing these scripts among multiple projects. I wonder if it's a good idea that we make a separate project for them and git submodule them in for use by each project. Then our `test.sh` can just import this file and set a few variables.

**Review by alastair — APPROVED** [review]
Nice :cactus:

File: Dockerfile (hunk 1 above)
Comment (paramsingh) [reply Dockerfile:1]: Yep.

File: README.md (hunk 2 above)
Comment (paramsingh) [reply README.md]: Not really, I'll remove it.

**Pull request merged**